### ISSUE TITLE
UrlSlugResolver does not resolve links with html in Node environment

### ISSUE THREAD

**Issue by makma** [Kentico/kontent-delivery-sdk-js#286]
### Brief bug description

UrlSlugResolver does not resolve links with `html` in Node environment

### Repro steps

1. Download [repo](https://github.com/Simply007/resolve-url-as-html)
2. `npm i`
3. `npm start`
4. check console

### Expected behavior

```
<h3><strong>Logotypes &amp; Badges</strong></h3>
<p><p>ahoj</p></p>
<p><p>ahoj</p></p>
<p><p>ahoj</p></p>
```

### Test environment

 - Node v12.16.1

### Additional context

Hint: https://github.com/Kentico/kontent-delivery-sdk-js/blob/master/lib/parser/adapters/parse5-rich-text.parser.ts

**Comment by Simply007** [comment]
@Enngage,
unfortunately, @huluvu21 found another use case, that does not work 😢 

When you place HTML that in top tag contains HTML attributes, they are striped in the output.
I have provided a showcase:
https://github.com/Simply007/resolve-url-as-html

If you want to keep it in separate issue - feel free to do so.

**Comment by Simply007** [comment]
There is a screenshot of the actual code (`context.item` is set since the code is using gatsby source plugin): 

<img width="484" alt="gatsby_resolver_code" src="https://user-images.githubusercontent.com/9218736/78143653-d93a0f80-742e-11ea-88fb-0f91eadb7645.png">

**Comment by Enngage** [comment]
Thank you @Simply007 . I've released a fix in `9.1.1` (https://github.com/Kentico/kontent-delivery-sdk-js/blob/master/CHANGELOG.md#911-2020-04-02) 

The only different is that both browser parser & parse5 automatically change the `download` attribute to `download=""` which is however equivalent so it should be ok.
